charts/base-cluster/templates/ingress/traefik.yaml

```diff
@@ -0,0 +1,89 @@
+{{- if eq .Values.ingress.provider "traefik" -}}
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: ingress-controller
+  namespace: ingress
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+    app.kubernetes.io/component: ingress
+spec:
+  chart:
+    spec: {{- include "base-cluster.helm.chartSpec" (dict "repo" "traefik" "chart" "traefik" "context" $) | nindent 6 }}
+  interval: 1h
+  driftDetection:
+    mode: enabled
+  {{- if .Values.monitoring.prometheus.enabled }}
+  dependsOn:
+    - name: kube-prometheus-stack
+      namespace: monitoring
+  {{- end }}
+  install:
+    crds: Skip
```

Comment:
Where do the CRDs get installed?

I know there is an extra Chart, but i can't find it here

Reply:
CRDs are explicitly not installed to encourage users to use standard APIs like Ingress and Gateway API instead of custom resources/configurations/annotations/... .